pycmr/queries.py

```diff
@@ -120,19 +120,105 @@ def version(self, version=None):
         self.params['version'] = version
         return self
 
-    def point(self, point=None):
+    def point(self, lon, lat):
         """
-        Set the point of the search we are querying
+        Set the point of the search we are querying.
+
+        :param lon: longitude to search at
+        :param lat: latitude to search at
+        :returns: Query instance
         """
 
-        if not point:
-            return
+        if not lat or not lon:
+            return self
```

Comment:
Return self or throw an exception? we have a few methods that just throw if they don't pass validation.